sov-modules/sov-modules-impl/accounts/src/call.rs

```diff
@@ -0,0 +1,80 @@
+use crate::Account;
+use crate::Accounts;
+use anyhow::Result;
+use borsh::{BorshDeserialize, BorshSerialize};
+use sov_modules_api::Signature;
+use sov_modules_api::{Address, CallResponse, PublicKey};
+
+pub const UPDATE_ACCOUNT_MSG: [u8; 32] = [0; 32];
+
+#[derive(BorshDeserialize, BorshSerialize, Debug, PartialEq)]
+pub enum CallMessage<C: sov_modules_api::Context> {
+    // Creates a new account.
+    CreateAccount,
+    // Updates a PublicKey for the corresponding Account.
+    // The sender must be in possession of the new PublicKey.
+    UpdatePublicKey(C::PublicKey, C::Signature),
+}
+
+impl<C: sov_modules_api::Context> Accounts<C> {
+    pub(crate) fn create_account(&mut self, context: &C) -> Result<CallResponse> {
+        self.exit_if_account_exist(context.sender())?;
+
+        let default_address = context.sender().to_address();
```

Comment:
non-blocking: Should we change `context.sender()` to be an address instead of a pubkey?

Reply:
I think we should have both, `context.sender_pub_key()` and `context.sender_addr()` but I will be able to populate the address only after we integrate `Accounts` module with the `DemoApp` so I will add it then (in 1 or 2 PRs).